core/src/sql/statements/define/access.rs

```diff
@@ -144,6 +141,38 @@ impl Display for DefineAccessStatement {
 				write!(f, " WITH JWT {}", ac.jwt)?;
 			}
 		}
+		// Always print relevant durations so defaults can be changed in the future
+		// If default values were not printed, exports would not be forward compatible
+		// None values need to be printed, as they are different from the default values
+		write!(f, " DURATION")?;
+		if self.kind.can_issue_grants() {
+			write!(
+				f,
+				" FOR GRANT {},",
+				match self.duration.grant {
+					Some(dur) => format!("{}", dur),
+					None => "NONE".to_string(),
+				}
+			)?;
+		}
+		if self.kind.can_issue_tokens() {
+			write!(
+				f,
+				" FOR TOKEN {},",
```

Comment:
Ideally, we wouldn't print the comma if we don't need to

Reply:
See [previous comment](https://github.com/surrealdb/surrealdb/pull/4119#discussion_r1627498518).